azure/services/async/async.go

```diff
@@ -132,20 +133,25 @@ func (s *Service) CreateResource(ctx context.Context, spec azure.ResourceSpecGet
 	}
 
 	// Create or update the resource with the desired parameters.
-	log.V(2).Info("creating resource", "service", serviceName, "resource", resourceName, "resourceGroup", rgName)
+	logMessageVerbPrefix := "creat"
```

Comment:
I love the more specific log messages here. (Good thing we don't have to i18n this.)